private/bufpkg/bufimage/bufimagetesting/bufimagetesting_test.go

```diff
@@ -120,7 +122,7 @@ func TestBasic(t *testing.T) {
 		"b/b.proto",
 		"a/a.proto",
 		"a/b.proto",
-		"b/b.proto",
+		"b/a.proto",
```

Comment:
This was an import cycle -- b/b.proto imported b/b.proto.

Now that we try to build a resolver from the image, to resolve any unrecognized fields, this gets noticed in the 
NewImageForProto call.